api/src/pcapi/routes/backoffice_v3/templates/providers/get/cgr.html

```diff
@@ -1,7 +1,7 @@
 {% import "components/links.html" as links %}
 {% import "providers/get/common.html" as common %}
 <turbo-frame id="cgr_frame">
-{{ common.build_provider_bar("cgr", form, dst, true) }}
+{{ common.build_provider_bar("cgr", form, dst, false) }}
```

Comment:
```suggestion
{{ common.build_provider_bar("cgr", form, dst) }}
```
Le but est à la fin d'enlever ce paramètre